packages/gatsby-image/src/index.js

```diff
@@ -48,18 +48,67 @@ const convertProps = props => {
   return convertedProps
 }
 
+/**
+ * Checks if fluid or fixed are art-direction arrays.
+ * @param props   {object}   The props to check for images.
+ * @param prop    {string}   Check for fluid or fixed.
```

Comment:
Please update the jsdoc with proper types. `prop` is an enum and `props` can be copied from `getImageSrcKey` below.

Example for enum:

```js
/**
 * @param prop {"fluid" | "fixed"}
 */
```